### PR TITLE
UI overhaul to fix various issues.

### PR DESCRIPTION
**Pull request by tkeeler**
This UI overhaul should fix a few of the various glitches. 
#240, #278, #257

While working noticed that link was still unfinished, so I merge the code I thought I had already submitted and got it working again.

### CODE REVIEW

**Comment by amshali** [comment]
@ProCynic , @jtessler can you help reviewing this?

File: app/views/tags/lhsNav.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <li><a href="/users/${_user.id}"> <strong>${_user.name}</strong></a></li>
   <li>
     <a href="/profilePicEditor" rel="tooltip" title="Change Picture">
-      <img class="thumbnail" src="@{Thumbnails.get120x120(_user.profile.profilePhoto.id)}" width="120" height="120" />
+      <img class="thumbnail" src="@{Thumbnails.get100x100(_user.profile.profilePhoto.id)}" width="100" height="100" />
```
Comment (amshali) [inline app/views/tags/lhsNav.html:5]: Why did you change this?

**Comment by amshali** [comment]
I personally do not like the current changes to the UI by this set. Let's hear other's opinion.

**Comment by jtessler** [comment]
It probably isn't necessary to add a 100x100 thumbnail when we could just resize the 120x120. I've also considered removing all thumbnails except 150x150, then resize where needed. I believe 150x150 is already small enough to reduce loading times; anything else is superfluous.

However, if you do want to keep the 100x100, mind adding a route to `conf/routes`?

I'll review the UI changes shortly.

**Comment by jtessler** [comment]
I added some screenshots to quickly compare the changes: http://imgur.com/a/x75MX

I personally like the fixed width style and keeping the right-side nav stationary while scrolling. However, the left-side nav seems too skinny now and its style doesn't match the right-side one (no shadows).

I'm not sure about moving the profile thumbnails outside of the post box, but it may grow on me.

File: app/views/tags/lhsNav.html (hunk 1 above)
Comment (tkeeler) [inline app/views/tags/lhsNav.html:5]: I added the 100x100 because using the bootstrap the left hand column would have to be 140 or 220. When set to 220 with the 120x120 image, the image looked totally lost. When using 120x120 on the 140 width, the image overflowed too much that it over took the column. 100x100 was the happy medium, and I made the new Thumbnail because that it an image that if it was resized would have to be resized on almost every page. It is a simple fix to go back to 120x120 or to continue using 100x100 with the 120x120 thumbnail it I need to.
